lib/parameters.py

```diff
@@ -54,6 +54,70 @@ def gen_avc_cqp_parameters(spec, profiles):
   params = gen_avc_cqp_variants(spec, profiles)
   return keys, params
 
+def gen_avc_icq_variants(spec, profiles):
+  for case, params in spec.iteritems():
+    for variant in copy.deepcopy(params.get("icq", [])):
+      uprofile = variant.get("profile", None)
+      cprofiles = [uprofile] if uprofile else profiles
+      for profile in cprofiles:
+        yield [
+          case, variant["gop"], variant["slices"], variant["bframes"],
+          variant.get("icq_quality"), profile
+        ]
+
+def gen_avc_icq_parameters( spec, profiles):
+  keys = ("case", "gop", "slices", "bframes", "icq_quality", "profile")
```

Comment:
Is "quality" (aka. target-usage (TU) or preset) parameter irrelevant for ICQ?  Can't we have both icq_quality and quality?